### PR TITLE
feat: Only active contacts and their number will be shown in Booking page 

### PR DESCRIPTION
**Pull request by AlAminKh03**
-  Only active contacts and their number will be shown in Booking page

### CODE REVIEW

**Comment by ahmad-ml** [comment]
good job !!

File: src/components/GuestDetailsInfo.tsx [hunk 1]
```diff
@@ -177,10 +177,13 @@ const GuestDetailsInfo = ({
                 <span>{option}</span>
               </>
             )}
-            options={contacts.map((contact) => ({
-              label: contact.name,
-              value: contact._id,
-            }))}
+            options={contacts
+              .filter((contact) => !contact.detactivatedAt)
+              .sort((a, b) => a.name.localeCompare(b.name))
```
Comment (ahmad-ml) [inline src/components/GuestDetailsInfo.tsx:182]: Sort is not working

**Review by ahmad-ml — CHANGES_REQUESTED** [review]
Just sort has some issue. Other than all ok.

**Comment by AlAminKh03** [comment]
Silly me, Sorry sir, I didn't notice that it has to be in the guestlookup (that is obvious) page , FIxed it . :"))

**Pull request closed without merging**